data/json/items/armor.json

```
@@ -5368,6 +5368,12 @@
     "storage": 3,
     "warmth": 30,
     "material_thickness": 3,
+	"snippet_category": [
+      {
+        "id": "patched",
```

This id isn't very descriptive, perhaps
```suggestion
        "id": "patched_jacket",
```

---

I opted for the single word id since it's just an additional snippet for jacket_jean rather than a completely new item, and that seemed to be in the convention of the t-shirts that have those extra snippets. I also didn't want to get it confused with any new, hypothetical "jacket_*" items in the future.

---

I totally missed that, thought it was a new item. You can resolve this conversation.